cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -5754,7 +5754,32 @@ action.cyences_send_email_action = 1
 action.cyences_notable_event_action.products = VPN
 action.cyences_notable_event_action.teams = SOC
 
-
+[Authentication - DUO Locked Out]
```

Comment:
Let's create new product **DUO** and will add alert under that product. alert name should be **DUO - User is Locked Out**

@VatsalJagani what is your thoughts?